core/mondoo-linux-inventory.mql.yaml

```diff
@@ -111,3 +115,46 @@ packs:
         title: Physical memory size
         mql: |
           command("free --mega | grep Mem | awk '{ print $2}'").stdout.trim + "M"
+      - uid: mondoo-linux-smbios-baseboard
+        title: SMBIOS baseboard (or module) information
+        mql: machine.baseboard { manufacturer version serial assetTag product }
+      - uid: mondoo-linux-smbios-bios
+        title: SMBIOS BIOS information
+        mql: machine.bios { vendor version releaseDate }
+      - uid: mondoo-linux-smbios-system
+        title: SMBIOS System information
+        mql: machine.system { sku serial family version product uuid manufacturer }
+      - uid: mondoo-linux-smbios-system
+        title: SMBIOS Chassis information
+        mql: machine.chassis { manufacturer serial version assetTag }
+      - uid: mondoo-linux-workstation-security-permissions-on-bootloader-config-metadata
+        title: Gather metadata on current bootloader config
+        filters: |
+          asset.family.contains('linux')
+          packages.where(name == /xorg|xserver|wayland/i).any(installed)
+        mql: |
+          if (file("/boot/grub/grub.cfg").exists) {file("/boot/grub/grub.cfg") {dirname basename permissions}}
+          if (file("/boot/grub2/grub.cfg").exists) {file("/boot/grub2/grub.cfg") {dirname basename permissions}}
+          if (file("/boot/grub/user.cfg").exists) {file("/boot/grub/user.cfg") {dirname basename permissions}}
+          if (file("/boot/grub2/user.cfg").exists) {file("/boot/grub2/user.cfg") {dirname basename permissions}}
+      - uid: mondoo-linux-workstation-security-secure-boot-is-enabled-metadata
+        title: Print out the results of 'mokutil --sb-state' for troubleshooting purposes
+        filters: |
+          asset.family.contains('linux')
+          packages.where(name == /xorg|xserver|wayland/i).any(installed)
+        mql: |
+          command('mokutil --sb-state').stdout
+      - uid: mondoo-linux-workstation-security-aes-encryption-algo-metadata
+        title: Print out the ciphersuite used for disk encryption
```

Comment:
Let's not use "print out the" form for these since these are the titles in the data queries tab for assets now

Reply:
@tas50 thank you for the feedback and change it.